docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1098,6 +1099,121 @@ team can more easilly read the messages.
 It is also possible to configure the persistence properties for a specific tenant, for example to set an alternate
 database hostname or database schema name.
 
+## 4.10 BulkData
+### 4.10.1 BulkData Export
+BulkData Export is implemented according to [FHIR r4 BulkData spec](http://hl7.org/fhir/uv/bulkdata/STU1/export/index.html).   
```

Comment:
```suggestion
BulkData Export is implemented according to [FHIR BulkDataAccess IG: STU1](http://hl7.org/fhir/uv/bulkdata/STU1/export/index.html).   
```
sorry - I realized I gave a bad edit before

Reply:
np, let me fix it.

Reply:
done